<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>403</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	background-color: #C0C0C0;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>441</width>
      <height>161</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: gray;
	border: 1px solid gray;
	color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>691</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>4-Function Calculator (plus exponent). See source for license.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Divide">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ExponentButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>x^y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Multiply">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DecimalButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Subtract">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ACButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Equals">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>293</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #FF8C00;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeSign">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastOpDisplay">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: gray;
	border: 1px solid gray;
	color: #ffffff;
}</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionfoo">
   <property name="text">
    <string>foo</string>
   </property>
  </action>
  <action name="actionbar">
   <property name="text">
    <string>bar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
